fuse_optimizers/src/optimizer.cpp

```diff
@@ -284,4 +284,16 @@ void Optimizer::notify(
   }
 }
 
+void Optimizer::injectCallback(
+  const std::string& sensor_name,
+  const std::set<ros::Time>& stamps,
+  const fuse_core::Transaction::SharedPtr& transaction)
+{
+  // We are going to insert a call to the derived class's transactionCallback() method into the global callback queue.
```

Comment:
*class'?

Reply:
No. I believe `class's` is correct.
https://english.stackexchange.com/questions/1073/what-is-the-correct-possessive-for-nouns-ending-in-s